test/test-applications/integrations/Samples.HttpMessageHandler/Program.cs

```diff
@@ -99,6 +99,32 @@ public static async Task Main(string[] args)
 
                 // sync http requests using HttpClient are not supported with WinHttpHandler
 #endif
+
+#if NETCOREAPP2_1 || NETCOREAPP3_0 || NETCOREAPP3_1
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    Console.WriteLine();
+                    Console.WriteLine("Sending async request with WinHttpHandler.");
```

Comment:
Maybe worth differentiating the "internal" (Http lib version) from the "public" (NuGet version) of WinHttpHandler?

```suggestion
                    Console.WriteLine("Sending async request with internal WinHttpHandler.");
```